projects/1.20/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,580 @@
+{
+	"key.toggleRingAttraction": "切换吸引戒指开/关",
+	"key.categories.gobber2": "戈伯",
+
+	"block.gobber2.gobber2_lucky_block": "戈伯幸运方块",
+	"block.gobber2.gobber2_lucky_block_deepslate": "深层戈伯幸运方块",
+	"item.gobber2.gobber2_lucky_block_deepslate": "深层戈伯幸运方块",
+	"block.gobber2.gobber2_lucky_block_nether": "下界戈伯幸运方块",
+	"item.gobber2.gobber2_lucky_block_nether": "下界戈伯幸运方块",
+	"block.gobber2.gobber2_lucky_block_end": "末地戈伯幸运方块",
+	"item.gobber2.gobber2_lucky_block_end": "末地戈伯幸运方块",
+
+	"block.gobber2.gobber2_ore": "戈伯矿石",
+	"item.gobber2.gobber2_ore": "戈伯矿石",
+	"block.gobber2.gobber2_ore_deepslate": "深层戈伯矿石",
+	"item.gobber2.gobber2_ore_deepslate": "深层戈伯矿石",
+	"item.gobber2.gobber2_ore.line1": "需要下界合金等级的镐子挖掘",
+
+	"item.gobber2.gobber2_glob": "戈伯簇",
+	"item.gobber2.gobber2_ingot": "戈伯锭",
+	"block.gobber2.gobber2_block": "戈伯块",
+	"item.gobber2.gobber2_block.line1": "方便存储戈伯锭",
+	"item.gobber2.gobber2_block.line2": "破坏时掉落完整方块",
+	"item.gobber2.gobber2_block.line3": "可以重新转换成锭",
+	"item.gobber2.gobber2_globette": "戈伯团",
+	"item.gobber2.gobber2_rod": "戈伯棒",
+	"item.gobber2.gobber2_foo": "戈伯煤炭",
+	"item.gobber2.gobber2_foo.line1": "比原版煤炭、木炭更好",
+	"item.gobber2.gobber2_foo.line2": "燃烧时间：%s",
+
+	"block.gobber2.gobber2_ore_nether": "下界戈伯矿石",
+	"item.gobber2.gobber2_ore_nether": "下界戈伯矿石",
+	"item.gobber2.gobber2_ore_nether.line1": "需要戈伯等级的镐子挖掘",
+	"item.gobber2.gobber2_glob_nether": "下界戈伯簇",
+	"item.gobber2.gobber2_globette_nether": "下界戈伯团",
+	"item.gobber2.gobber2_ingot_nether": "下界戈伯锭",
+	"block.gobber2.gobber2_block_nether": "下界戈伯块",
+	"item.gobber2.gobber2_rod_nether": "下界戈伯棒",
+	"item.gobber2.gobber2_foo_nether": "下界戈伯煤炭",
+
+	"block.gobber2.gobber2_ore_end": "末地戈伯矿石",
+	"item.gobber2.gobber2_ore_end": "末地戈伯矿石",
+	"item.gobber2.gobber2_ore_end.line1": "需要下界戈伯等级的镐子挖掘",
+	"item.gobber2.gobber2_glob_end": "末地戈伯簇",
+	"item.gobber2.gobber2_globette_end": "末地戈伯团",
+	"item.gobber2.gobber2_ingot_end": "末地戈伯锭",
+	"block.gobber2.gobber2_block_end": "末地戈伯块",
+	"item.gobber2.gobber2_rod_end": "末地戈伯棒",
+	"item.gobber2.gobber2_foo_end": "末地戈伯煤炭",
+
+	"item.gobber2.dragon_star": "龙之星",
+	"item.gobber2.dragon_elytra": "龙之鞘翅",
+	"item.gobber2.dragon_elytra.tip": "注：仅用于合成，并非真正的鞘翅",
+
+	"item.gobber2.gobber2_excavator.line1": "一把能破坏3x3区域的铲子",
+	"item.gobber2.gobber2_hammer.line1": "一把能破坏3x3区域的锤子",
+	"item.gobber2.gobber2_paxel.line1": "结合了镐、斧和锹",
+	"item.gobber2.gobber2_tree_axe.line1": "会尝试砍伐整棵树",
+	"item.gobber2.gobber2_end.unbreakable": "无法破坏的工具",
```

是工具不会被损坏、还是工具无法破坏其他方块？

```suggestion
	"item.gobber2.gobber2_end.unbreakable": "不会损坏的工具",
```
此类问题请全文通查并修改

---

参照 Minecraft 原版翻译“Unbreakable”→“无法破坏”